dataclients/kubernetes/definitions/testdata/validation/route-with-invalid-backend-with-path.log

```diff
@@ -0,0 +1 @@
+backend address \"http://example.com/foo\" contains path\nbackend address \"http://example.com/foo/bar\" contains path\nbackend address \"/foo\" contains path\nbackend address \"/foo/bar\" contains path
```

Comment:
minor: add newline here for :stop_sign: at the end of this file to disappear.

Reply:
It's like this because I still didn't make tests work.